SaintCoinach/ex.json

```diff
@@ -4944,6 +4944,46 @@
         }
       ]
     },
+    {
+      "sheet": "Frontline04",
+      "definitions": [
+        {
+          "name": "Level[1]",
```

Comment:
Maybe spawn positions for the 3 GCs, plus positions of ice or nodes?

Reply:
Happily change if we can confirm it ?